extensions/mongodb-client/runtime/src/main/java/io/quarkus/mongodb/runtime/graal/MongoClientSubstitutions.java

```diff
@@ -260,4 +260,9 @@ final class JnaNativeSubstitutions {
     public static String getWebStartLibraryPath(final String libName) {
         return null;
     }
+
+    @Substitute
+    public static void register(Class<?> cls, String libName) {
```

Comment:
Shouldn't we downgrade to the version that worked until we upgrade MongoDB? That's what I was planning to do today.

Reply:
I suppose we could just revert https://github.com/quarkusio/quarkus/pull/25114.

Reply:
Yeah, that would have my preference.

Reply:
Okay: https://github.com/quarkusio/quarkus/pull/25717

Reply:
Yes, I prefere to revert it.

This library is an issue as it mandates an enterprise Mongo version to be testable.
Maybe we should conditionally include a substitution for mongo-crypt in case the library is not in the classpath and remove it from the extension dependencies.